src/index.js

```diff
@@ -292,7 +265,10 @@ class Dropzone extends React.Component {
       ...props // eslint-disable-line prefer-const
     } = rest;
 
-    const { isDragActive, isDragReject } = this.state;
+    const { draggedFiles } = this.state;
+    const isMultipleAllowed = multiple || draggedFiles.length <= 1;
```

Comment:
I know it's a nit peak but you're accessing `length` property 3 times in a row. Should it be extracted, you think?